### PR TITLE
Dev wine in trail

### PR DESCRIPTION
**Pull request by john-baxter**
The wine is now included in the trail, can be collected from the trail and added to the hand. There is no scoring mechanism yet, this is going to be the next thing to be added.

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -47,13 +47,21 @@
...
+
+NUMBER_OF_EACH_STANDARD_FOOD_TOKEN = 9
+NUMBER_OF_EACH_SPECIAL_FOOD_TOKEN = 5
+"""As per the standard rules of the game, there are this many of each  
 food token.
 """
 
 
-TOKENS_FOR_TRAIL = {i : NUMBER_OF_EACH_FOOD_TOKEN for i in FOOD_TYPES}
+STANDARD_TOKENS_FOR_TRAIL = {i : NUMBER_OF_EACH_STANDARD_FOOD_TOKEN for i in STANDARD_FOOD_TYPES}
```
Comment (anaandresarroyo) [inline constants.py]: `i` is typically used to represent integers. I suggest changing to `food` for clarity.

File: constants.py [hunk 2]
```diff
@@ -63,6 +71,25 @@
...
+"""The standard number of each special food token
+
+A dictionary showing how many of each special food token will be 
+used as per the rules.
+Keys are foods as strings
+Values are integers
+"""
+
+
+TOKENS_FOR_TRAIL = dict(STANDARD_TOKENS_FOR_TRAIL, **SPECIAL_TOKENS_FOR_TRAIL)
```
Comment (anaandresarroyo) [inline constants.py:84]: You might be interested to know that from Python 3.9 (released just a few days ago on the 17th Oct 2020), the combining of two dictionaries has been simplified. See https://stackoverflow.com/questions/38987/how-do-i-merge-two-dictionaries-in-a-single-expression-in-python-taking-union-o

**Review by anaandresarroyo — APPROVED** [review]
Excellent. Very well done!

File: constants.py (hunk 2 above)
Comment (john-baxter) [reply constants.py:84]: Yes, I actually read some of this page earlier.

Comment (anaandresarroyo) [reply constants.py:84]: Note how the stack overflow link suggests `{**STANDARD_TOKENS_FOR_TRAIL, **SPECIAL_TOKENS_FOR_TRAIL}` with `**` in the first one too. That seems to be required when using `{}` but not when using `dict()` (although it does work too if you use the `**` in the first argument of the `dict()`).

**Pull request merged**